week-4/Homework/mandatory/3-water-bottle.js

```diff
@@ -20,19 +20,36 @@ You have to implement the missing features according to the specification.
 // Here is your starting point:
 let bottle = {
   volume: 0,
-  fillUp: function() {
-    // calling this function should pour your bottle full (volume = 100);
+  fillUp: function () {
+    while (this.volume < 100) {
+
+      this.volume++;
+    }
+    console.log(this.volume);
   },
+  // calling this function should pour your bottle full (volume = 100);
   pour: function () {
+    this.volume1 = this.volume + 10
+    console.log(this.volume1);
+
     // calling this function should increase your bottle volume by 10 unit;
   },
-  drink: function() {
+  drink: function () {
+    this.volume = this.volume - 10
+    console.log(this.volume);
     // calling this function should decrease your bottle volume by 10 unit;
   },
   isFull: function () {
     // this function should return true if your bottle is empty;
+    if (this.volume === 100) {
+      return true
+    }
+
   },
-  isEmpty: function() {
+  isEmpty: function () {
+    if (this.volume === 0) {

```

Comment:
if (this.volume===100)
the expression itself will return true or false 
so do you need the condition
```return this.volum ===100```